### PR TITLE
fix: apsara 0.8.0 issue fixes

### PR DESCRIPTION
**Pull request by abhishekv24**

### CODE REVIEW

**Review by ravisuhag — APPROVED** [review]

File: packages/apsara-ui/package.json [hunk 1]
```diff
@@ -52,10 +52,18 @@
         "@radix-ui/react-avatar": "^1.0.0",
         "@radix-ui/react-dropdown-menu": "^1.0.0",
         "@tanstack/react-table": "8.5.13",
+        "@rc-component/mini-decimal": "1.0.1",
         "react-query": "3.39.2",
         "react-virtual": "^2.10.4",
         "rc-pagination": "^3.1.17",
         "antd": "^4.16.13",
```
Comment (StewartJingga) [inline packages/apsara-ui/package.json:59]: So if we remove this, we would need to import lots of packages ?

Does that mean Apsara depend on libraries that are `antd`'s dependencies?

File: packages/apsara-ui/src/Button/Button.types.tsx [hunk 2]
```diff
@@ -1,8 +1,61 @@
...
+    | "right"
+    | "bottom"
+    | "topLeft"
+    | "topRight"
+    | "bottomLeft"
+    | "bottomRight"
+    | "leftTop"
+    | "leftBottom"
+    | "rightTop"
+    | "rightBottom";
```
Comment (StewartJingga) [inline packages/apsara-ui/src/Button/Button.types.tsx:57]: Just a question, are these all antd props?

File: packages/apsara-ui/package.json (hunk 1 above)
Comment (abhishekv24) [reply packages/apsara-ui/package.json:58]: removed this dependency as we were using only one type from there.

File: packages/apsara-ui/src/Button/Button.types.tsx (hunk 2 above)
Comment (abhishekv24) [reply packages/apsara-ui/src/Button/Button.types.tsx:57]: Yes, ButtonProps and TooltipPlacement types were being imported from antd here.

**Comment by ravisuhag** [comment]
@StewartJingga @abhishekv24 i am merging this PR. if they can other small issues we can cover them in another one.

**Pull request merged**